app.py

```diff
@@ -0,0 +1,148 @@
+import json
+from flask import Flask, redirect, render_template, flash, session, request
+from models import User, connect_db, db
+import os
+from secret import SECRET_KEY, OPEN_CHARGE_MAP_KEY, MAP_KEY
+from forms import SignUpForm, LoginForm
+from sqlalchemy.exc import IntegrityError
+import requests
+
+app = Flask(__name__)
+
+app.config["SECRET_KEY"] = os.environ.get('SECERT_KEY', SECRET_KEY)
+app.config["SQLALCHEMY_DATABASE_URI"] = os.environ.get(
+    'DATABASE_URL', "postgresql://localhost/chargR?user=postgres&password=postgresql")
```

Comment:
The DB password should come from the same place as other sensitive data. Please create a new constant and treat it the same way as the `SECRET_KEY`, `MAP_KEY`, etc.